### DISCUSSION TITLE
Can Technic hub receive Bluetooth Signal from a Pi?

### DISCUSSION THREAD

**Discussion by GenghisHan-op** [pybricks/support#1930] [General]
I read the document and know that the hubs can send and receive bluetooth signal from each other, but can they receive from a Pi? For example, the pi will act as a controller and send command to the hub to control the motor via bluetooth. If it can, please tell me how.
Thank you.

**Comment by BertLindeman** [comment]
Hi,

Did you read discussion #1599 quite a broad discussion.
In that discussion is  [A Python implementation of the Pybricks connectionless Bluetooth messaging protocol](https://github.com/fkleon/pybricks-ble) 

Bert

**Reply by GenghisHan-op** [reply]
Thank you very much, this is exactly what I am looking for. I'll give it a try.
